lib/lhm/chunker.rb

```diff
@@ -61,6 +67,14 @@ def throttle_seconds
       @throttle / 1000.0
     end
 
+    def select_query(offset, columns_to_be_selected = nil)
+      columns_to_be_selected ||= columns
+      "select #{ columns_to_be_selected } from `#{ origin_name }` " +
+        "where `#{origin_primary_key}` > #{offset} " +
+        "and `#{origin_primary_key}` <= #{@limit} " +
```

Comment:
Just need clarification on this condition, Should it be `offset+limit` ?

Reply:
@Rajeshguna Here `@limit` denotes `max(id)` and not batch limit. Batch limit is denoted by `@stride`. Hence it is `id <= @limit ORDER BY id ASC LIMIT @stride` Using the same terms as present in gem already.

Reply:
Got it, Any particular advantage on having this condition as we have limit in the query?

Reply:
Couldn't think of any problem by not having this condition, replicated the existing logic of copying the records only between MIN and MAX ids in order to be consistent.